ui_framework/components/prompt_for_items/prompt_for_items.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import classnames from 'classnames';
+
+import { KuiButtonIcon, KuiLinkButton } from '../button';
+import { KuiPromptForItemsMessage } from  './prompt_for_items_message';
+
+export function KuiPromptForItems({ itemType, promptMessage, promptButtonText, addHref, className, ...rest }) {
+  const classes = classnames('kuiPromptForItems', className);
+  return (
+    <div className={classes} {...rest} >
+      <KuiPromptForItemsMessage>
+        {promptMessage}
+      </KuiPromptForItemsMessage>
+
+      <div className="kuiPromptForItems__actions">
+        <KuiLinkButton
+          icon={<KuiButtonIcon type="create" />}
+          aria-label={`Add a new ${itemType}`}
+          className="testClass1 testClass2"
+          data-test-subj={`addNewPromptButton`}
+          buttonType="primary"
+          href={addHref}
+        >
```

Comment:
> But now that I think about it, the current PR fits this description already, doesn't it?

Very close, except that is requests an `addHref` and preformats the add button so that it has a + icon.  Only the text differs.

`KuiEmptyTablePrompt`?  `EmptyPrompt` is a bit vague, but empty table prompt, sgtm.